packages/global/components/blocks/pmg-events.marko

```diff
@@ -1,4 +1,4 @@
-$ const { i18n } = out.global;
+$ const { i18n, site, config } = out.global;
```

Comment:
Is `site` used anywhere? I see it got added here but there doesn't appear to be a use of it also added.